modules/acquia_telemetry/README.md

```diff
@@ -0,0 +1,10 @@
+# Acquia Telemetry
+
+This module sends anonymous data about Acquia product usage to Acquia
+for product development purposes.
+
+No private information will be gathered. Data will not be used for
+marketing and will not be sold to any third parties.
+
+Telemetry is opt-in only and can be disabled at any time by uninstalling
```

Comment:
The only concern I have about having the `acquia_telemetry` installation _constitute_ opting into telemetry is that this is a consciously short-term solution. Will replacing it with our permanent solution be more complicated by this approach? Should we use a State value instead?

Reply:
To me, tying Telemetry to installation seems to simplify things. It provides a simple interface for enabling/disabling the behavior, and we can be sure that no bugs cause accidental capturing of data.

Reply:
I agree with @grasmash on this one. Assuming it is disabled by default, installing the module is a very clear opt-in action, and we don't need to further complicate it.